examples/user-interface/text-canvas-font.html

```diff
@@ -58,7 +58,7 @@
         app.assets.add(fontAsset);
         app.assets.load(fontAsset);
         app.assets.on('load:' + fontAsset.id, function () {
-            app.start();
+            
         });
```

Comment:
You might as well delete this event handler if it doesn't do anything.

Reply:
yep, done